app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
```

Comment:
I don't think we are using this controller anywhere, don't push anything which we are not using.